packages/grafana-toolkit/package.json

```diff
@@ -72,7 +72,7 @@
     "@typescript-eslint/parser": "5.36.2",
     "axios": "^0.26.1",
     "babel-jest": "27.5.1",
-    "babel-loader": "^8.2.5",
+    "babel-loader": "^9.1.0",
```

Comment:
Thanks for taking a look at this @joeyorlando. I'm not sure it is necessary to major version bump to resolve this issue for plugins. Looking at the [babel-loader@8.2.5 package.json](https://github.com/babel/babel-loader/blob/v8.2.5/package.json#L14) it can already resolve the dependency of `loader-utils` to 2.0.4. 

Changing this to `^8.3.0` would likely solve it for any dev updating to latest toolkit (so long as their plugin doesn't declare any other webpack loaders that pull in an old version of `loader-utils`). Any new plugin scaffolded with toolkit will already receive the patched version.